mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -76,10 +77,12 @@ def runner(self, drive):
         while True:
             sleep(2)
             if platform.system() == 'Windows':
-                out = os.popen('dir %s' % drive[0]).read()
+                proc = Popen(["dir", drive[0]], stdin=PIPE, stdout=PIPE, stderr=PIPE)
+                out = proc.stdout.read()
```

Comment:
I think it is better to use `communicate()` method here, because it will collect output and wait for process termination